app/models/setting/content.rb

```diff
@@ -137,6 +137,9 @@ def self.load_defaults
                  N_("If hosts fail to register because of duplicate DMI UUIDs " \
                     "add their comma-separated values here. Subsequent registrations will generate a unique DMI UUID for the affected hosts."),
                  [], N_('Host Duplicate DMI UUIDs')),
+        self.set('host_profile_assume', N_("Allow new Host registrations to assume registered profiles with matching hostname " \
+                    "as long as the registering DMI UUID is unique."),
+                 true, N_('Host Profile Assume')),
```

Comment:
Trying to understand this working.   "as long as registering DMI uuid is unique".  Do you mean 'unique other than the host its taking' ?  I assume it already would take the host's profile if its uuid matched that, right?

Reply:
Correct! It's kind of a strange behavior to articulate. Let me know if you have a suggestion

Reply:
"as long as the registering DMI UUID is not used by another host" ?

Reply:
I'm good with that, updating